### PR TITLE
Server setup and basic index.html file

### PR DESCRIPTION
**Pull request by ChuaJ**
Hey Matt! Could you take a look at my code when you get a chance? Thanks!

### CODE REVIEW

**Comment by ChuaJ** [comment]
I also made some changes to the data.sql file and it's marking them as conflicts, so I can resolve those once you approve the code.

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+app.put("/:id/reviews", (req, res) => {
+  connection.query(`UPDATE reviews SET votes = ${req.body.votes}, helpfulness = ${req.body.helpfulness}
+  WHERE id = ${req.body.id}`, (err, results) => {
+    err
+    ? res.status(500).json(err)
+    : res.status(204).end();
+  });
+});
+
+app.listen(3001, () => console.log("Express server listening on port 3001"));
```
Comment (xueccc) [inline server/server.js]: add new line at the end

Comment (xueccc) [reply server/server.js]: Very nice! Good work on the queries! Lines 20, 39 & 40 are a little hard to follow, maybe reformat to make it more readable?
